build.gradle.kts

```diff
@@ -135,7 +135,7 @@ dependencies {
   api(gradleApi())
   api("com.squareup:javapoet:1.11.1")
   implementation("io.github.microutils:kotlin-logging:1.6.26")
-  implementation("com.squareup.okhttp3:okhttp")
+  implementation("com.squareup.okhttp3:okhttp:3.12.0")
```

Comment:
adding the version here is not necessary because there is a resolution at https://github.com/mkobit/jenkins-pipeline-shared-libraries-gradle-plugin/blob/bbc5b5a93ebad7d5653cf2e29545ab19273f76f8/build.gradle.kts#L123 that sets it for all `com.squareup.okhttp3` groups

Reply:
I failed to compile it without it.
I added the plugin as a includeBuild. 
Split from master (not sure that is the rigth branch) 
And i saw there was a class DependencyInfo that use to manage the dependency by in this version is missing.

Reply:
I am also having issue when running the generateIntegrationTestResources - it looks like the plugin add the folder under build as source and intelij get confuse cause we do not tell him that those are generated sources , i am investigating it ...

Reply:
The composite build thing is odd.

I recently got rid of the `buildSrc/.../DependencyInfo.kt` as I have started to not like the pattern.

The build passes CI though, so I think i'll merge it and make a few additional changes before releasing.

Reply:
Submitted https://github.com/gradle/gradle/issues/9508 for the composite build issue

Reply:
@mkobit are you a robot ? already reproduce it ? ?!!? amaizing!!!

Reply:
I did not had time to investigate the  intellij issue - hope to do it tonight ... 

think the plugin should configure the Inteliij  idea module    generatedSourceDirs 
Like in here :
https://docs.gradle.org/current/dsl/org.gradle.plugins.ide.idea.model.IdeaModule.html

idea {

 module { 
//and hint to mark some of existing source dirs as generated sources
generatedSourceDirs += file('some-extra-source-folder') }} 

The issue I see after running the Jenkins generation for integration tests is all the module's content  disappear from the project structure  - To fix it I need to run clean task and module content is back ...

I ll try to find the root and  I ll open a new pull request if it's ok ...

Also I wanted to ask when the release is planed for ? 

Thanks

Reply:
I don't think I fully understand the IntelliJ issue. I just import the project as a module by clicking on `settings.gradle.kts`.

I'll try and release a new version this week.